packages/mui-material/src/Breadcrumbs/BreadcrumbCollapsed.js

```diff
@@ -35,18 +35,39 @@ const BreadcrumbCollapsedIcon = styled(MoreHorizIcon)({
  * @ignore - internal component.
  */
 function BreadcrumbCollapsed(props) {
+  const { components, componentsProps, ...otherProps } = props;
```

Comment:
```suggestion
  const { components, componentsProps, ...other } = props;
```

For consistency with the other components.